common/es_proxy.py

```diff
@@ -15,6 +16,10 @@ def add_basic_es_arguments(parser):
               help='Elasticsearch password', env_var='ES_PASSWORD')
     group.add('--index-name',  dest='index_name', required=True,
               help='Elasticsearch index name')
+    group.add('--aws-access-key',  dest='aws_access_key',
+              help='If AWS, an access key is required') 
+    group.add('--aws-secret-key',  dest='aws_secret_key',
+              help='If AWS, a secret key is required')                     
```

Comment:
Can also use `env_var='AWS_FOO_BAR'` if you want to set up the environment => variable

Reply:
OK - is this still a useful param to have in here from a self-documenting perspective so that folks understand that such vars are required, or would you recommend leaving it out?

Reply:
It is definitely a useful param. I usually declare a lot of things to be params, but then add defaults and ENVs so that it can be configured easily from lots of sources.

Reply:
- https://github.com/cfpb/ccdb-data-pipeline/blob/main/complaints/ccdb/push_s3.py#L98

Reply:
Ah - now I see what you're saying. Thanks.